tests/vppoutputencode.cpp

```diff
@@ -103,10 +107,15 @@ static void setEncodeParam(const SharedPtr<IVideoEncoder>& encoder,
     encVideoParams.intraPeriod = encParam->intraPeriod;
     encVideoParams.ipPeriod = encParam->ipPeriod;
     encVideoParams.rcParams.bitRate = encParam->bitRate;
+    encVideoParams.rcParams.targetPercentage = encParam->targetPercentage;
```

Comment:
but libyami also assigned the target_percentage even the value is equal to the DEFAULT.
`rateControl->target_percentage = m_videoParamCommon.rcParams.targetPercentage;`
so I think I should modify the libyami's code to protect the driver's default value.